### PR TITLE
edit login.controller and server

### PR DESCRIPTION
**Pull request by yosrada**

### CODE REVIEW

**Review by MuhammadAw90 — APPROVED** [review]

**Comment by MuhammadAw90** [comment]
@Karyum I believe you need this pull request too

**Comment by Karyum** [comment]
@MuhammadAw90 Yup we are just discussing on how to go about this safely, Thank you 😄

**Review by Karyum — APPROVED** [review]
### [Speed merging for the following reason](https://github.com/WebAhead-cs2/caps-look/pull/70#issuecomment-1428709106)

Good job @yosrada

File: server.js [hunk 1]
```diff
@@ -16,6 +16,12 @@ app.use(
     statusLevels: true
   })
 )
+app.use(
+  cors({
+    credentials: true,
+    origin: ['http://localhost:3000']
+  })
+)
```
Comment (tawfiknasser) [inline server.js:24]: Delete the same middleware from line 32/38

**Pull request merged**